sympy/geometry/line.py

```diff
@@ -1180,7 +1254,7 @@ def __new__(cls, *args, **kwargs):
                 if len(args) > 1:
                     p2 = args[1]
                 else:
-                    p2 = None
+                    p2=None
```

Comment:
Please keep the spaces around =
For reference check https://www.python.org/dev/peps/pep-0008/